.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: node_js
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - g++-4.8
+cache:
+  directories:
+    - node_modules
+env:
+  global:
+    - CXX=g++-4.8
+  matrix:
+    - NPM_VERSION=2
+    - NPM_VERSION=3
+matrix:
+  fast_finish: true
+node_js:
+  - "0.10"
+  - "0.12"
+  - "4"
+  - "5"
```

Comment:
@jarena and @bheiskell I've added Node 5 to the build matrix. Please let me know if that would be an issue given our other policies.